src/main/webapp/WEB-INF/resources/ApplicationResources_pt.properties

```diff
@@ -83,23 +84,26 @@ label.no.statutes.defined = Ainda n\u00E3o foram definidos quaisquer estatutos.
 label.noResultsFound=N\u00E3o Foram Encontrados Resultados
 label.period = Per\u00EDodo
 label.org.fenixedu.academic.dto.accounting.AnnulAccountingTransactionBean.amountWithAdjustment = Valor (&euro;)
-label.org.fenixedu.academic.dto.accounting.AnnulAccountingTransactionBean.paymentMethod = Método de Pagamento
-label.org.fenixedu.academic.dto.accounting.AnnulAccountingTransactionBean.paymentReference = Referência de Pagamento
+label.org.fenixedu.academic.dto.accounting.AnnulAccountingTransactionBean.paymentMethod = M\u00E9todo de Pagamento
```

Comment:
Is the escaping intentional or an automatic formatting mistake from the IDE?

Reply:
Still waiting for you to fix this clusterfu** with the bundle files 😜

Reply:
JDK 9+ whenever you want :rofl: